samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,179 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
 
 See also the :ref:`nrf_cloud_mqtt_multi_service_test_counter`.
 
+
+.. _nrf_cloud_mqtt_multi_service_hardcoded_certs:
+
+Provisioning with hard-coded CA and device certificates
+=======================================================
+
+The :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` allows you to use hard-coded CA and device certificates stored in unprotected program memory for connecting to nRF Cloud.
+
+.. important::
+   The :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option is not secure, and should be used only for demonstration purposes!
+   This setting stores your device's private key in unprotected program memory, making your device vulnerable to impersonation.
```

Comment:
```suggestion
   Because this setting stores your device's private key in unprotected program memory, using it makes your device vulnerable to impersonation.
```